crates/ty_server/tests/e2e/main.rs

```diff
@@ -226,15 +225,13 @@ impl TestServer {
         let init_params = InitializeParams {
             capabilities,
             workspace_folders: Some(workspace_folders),
-            // TODO: This should be configurable by the test server builder. This might not be
-            // required after client settings are implemented in the server.
             initialization_options: initialization_options
-                .map(|options| json!({ "settings": options})),
+                .map(|options| serde_json::to_value(options).unwrap()),
```

Comment:
Maybe use `expect` or `or_else` to give the test author more context about what's going wrong.

Reply:
I'm using `.context`